src/paymentservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/paymentservice/BamboraConfiguration.java

```diff
@@ -25,16 +24,17 @@
 
     @RequestMapping(value = "/bamboraconfiguration", method = RequestMethod.GET)
     public BeanstreamEndpointResponse singlepaymenturl() {
-		// TODO - Complete logging once available.
         // Make sure autowired params are not null (May be missing from configMap)
         if (StringUtils.isEmpty(approved) || StringUtils.isEmpty(declined) || StringUtils.isEmpty(error)) {
-			return new BeanstreamEndpointResponse("", "", "", PaymentServiceConstants.PAYMENT_SERVICE_RESP_MSG_FAIL,
-					PaymentServiceConstants.PAYMENT_SERVICE_FAILURE_CD);
-		}
-        else
+            logger.error("CRC endpoints are not set properly in the config. Failed to read the properties. ");
```

Comment:
I would use logging.fatal for this and all major errors.

Reply:
changed error to fatal